### PR TITLE
resources and navigation

### PR DESCRIPTION
**Pull request by ralfhandl**
Part of #43:
- no implicit top-level resources
- `type Query` defines the top-level readable resources
- `type Mutation` defines the top-level modifiable resources

### CODE REVIEW

File: docs/rapid-pro-rsdl.md [hunk 1]
```diff
@@ -1,92 +1,90 @@
...
 
-
 ```graphql
-type Person {
-    UserName: String! @RapidID
-    FirstName: String
-    LastName: String
-    MiddleName: String
-    Age: BigInt
+type company {
```
Comment (wtrocki) [inline docs/rapid-pro-rsdl.md:28]: Any reason for small letter?

Comment (ralfhandl) [reply docs/rapid-pro-rsdl.md:28]: This is the example service we are using for the intro page: https://jetsons.azurewebsites.net/$metadata

File: docs/rapid-pro-rsdl.md [hunk 2]
```diff
@@ -161,7 +159,37 @@ Guid
...
+
+type employee {
+    id: Int32! @RapidID
+    firstName: String
+    lastName: String
+    title: String
+}
+```
+
+If these related types are not listed in the `Query` type, they can only be accessed indirectly:
```
Comment (wtrocki) [inline docs/rapid-pro-rsdl.md:180]: This is sooo clever!

**Comment by wtrocki** [comment]
THE best PR and proposition I have seen so far! Very simple and minimalistic. 
Really easy to understand and start building schemas. What we need is to refine operations and filters :)

File: docs/rapid-pro-rsdl.md [hunk 3]
```diff
@@ -1,92 +1,91 @@
...
+```
 
-> `type Person` => GET `/v4/Rapid/Person`
+Each field of `Query` defines a top-level resource that can be reached via a URI ending in the same name:
 
-Executing `GET` should return the list of the Persons in the database
+> GET `/v4/rapid/company`
 
-### Customization of the URI scheme
+will return the company this service is about.
```
Comment (wtrocki) [inline docs/rapid-pro-rsdl.md:62]: remove .

File: docs/rapid-pro-rsdl.md [hunk 4]
```diff
@@ -1,92 +1,91 @@
...
+
+By default RAPID is not exposing operations that can modify resources.
+These are enabled via the special `type Mutation`:
 
 ```graphql
 type Mutation {
-    ## Allows to perform write operation to the person object
-    Person: Person
+    # Allows update operation to the company resource
+    company: company @RapidUpdate
```
Comment (wtrocki) [inline docs/rapid-pro-rsdl.md:80]: This looks strange to me as types are usually starting with upper case.

Comment (ralfhandl) [reply docs/rapid-pro-rsdl.md:80]: Created #78

**Pull request merged**